onnxruntime/core/providers/cuda/rnn/rnn.h

```diff
@@ -30,9 +33,11 @@ class RNN final : public CudnnRnnBase<T> {
     // ONNX B layout is Wb, Rb, mapping to RNNLinLayerMatrixParams
     // the linLayerID is 0, 1, we can reuse it from W_lin_layer_id & R_lin_layer_id
 
-    ORT_THROW_IF_ERROR(CudnnRnnBase<T>::CacheCudnnRnnWeights(info));
+    //ORT_THROW_IF_ERROR(CudnnRnnBase<T>::CacheCudnnRnnWeights(info));
```

Comment:
Why do we not cache Rnn weights anymore ?

Reply:
While updating the change I disabled the caching. Reenabling it.